lib/extended-css.js

```diff
@@ -143,12 +145,23 @@ var ExtendedCss = function (styleSheet) { // jshint ignore:line
         affectedElement.node.style.cssText = affectedElement.originalStyle;
     };
 
+    /**
+     * This flag indicates that we should enable performance debugging for all selectors.
+     */
+    var globalDebugFlag = false;
+
     /**
      * Applies specified rule and returns list of elements affected
      * @param rule Rule to apply
      * @returns List of elements affected by this rule
      */
     var applyRule = function (rule) {
+        var debug = globalDebugFlag || rule.debug; 
```

Comment:
1. [ ] I'd like the global flag to be enabled with a rule as well.

So that filters maintainers could have that rule in the user filter and notice if extcss takes too much time or called too often.